analysis.py

```diff
@@ -146,6 +151,29 @@ def gpgllParse(self, gpgll, line):
          if self.verifyChecksum(line, gpgll[5]):
              return gpgll
 
+
+ 	 '''
+
+    	 GPVTG properties
+    
+   	 gpvtg[0] - "Track Made Good and Ground Speed"
+  	 gpvtg[1] - True track made good
+   	 gpvtg[2] - Magnetic track made good
+   	 gpvtg[3] - Ground speed, knots
+   	 gpvtg[4] - ground speed, Kimometers per hour
+   	 gpvtg[5] - Checksum
+
+   	 '''
+
+    def gpvtgParse(self, gpvtg, line):
+         gpvtg[1] = line[1] + line[2]
+         gpvtg[2] = ("0.00" if line[3] == "" else line[3])+line[4]
+         gpvtg[3] = line[5] + line[6]
+         gpvtg[4] = line[7] + line[8]
+	 gpvtg[5] = line[9]
```

Comment:
line 173: move this over a few spaces so it's in line with the others, it works fine but python's weird with spacing sometimes and this might be an issue with some versions of the python interpreter